File: PlayMath-SPA/src/app/_models/Question.ts
import { Category } from "./Category";

export interface Question {
    id: number;
    title: string;
    body: string;
    categoryName: string;
    categoryId: number;
    postDate: Date;
    questioner: string;
    viewed: number;
    isSolved: boolean;
    category: Category;
    answerCount: number;
}

File: PlayMath.API/models/Question.cs
using System;
using System.Collections.Generic;

namespace PlayMath.API.Models
{
    public class Question
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Body { get; set; }
        public DateTime PostDate { get; set; }
        public int Viewed { get; set; }
        public bool IsSolved { get; set; }
        public bool IsDeleted { get; set; }
        public ArticleCategory Category { get; set; }
        public User QuestionBy { get; set; }
        public ICollection<Answer> Answers { get; set; }
    }
}